### PR TITLE
Add AI Explainability meeting details

### PR DESCRIPTION
**Pull request by ruivieira**
## Description

Add details on the regular AI explainability community call.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by ruivieira** [comment]
@taneem-ibrahim @LaVLaS @anishasthana ptal. Thank you!

File: wg-xai/README.md [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+
+* Working Group Meeting: monthly on the third Wednesday @ 11am Eastern Time ([Meeting Link](https://meet.google.com/pwb-trdw-tsq))
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1iY6pz8EtLhxabkwPmqsVQ1MOC7mZJSMuEjrbK7F0q6U/edit?usp=sharing).
 
 ## Organizers
 
 * Rebecca Whitworth
 * Tommaso Teofili
-* Rui Vieira
+* [Rui Vieira](https://github.com/ruivieira)
```
Comment (anishasthana) [inline wg-xai/README.md:26]: Could you add similar GitHub links for the other organizers?

File: wg-xai/README.md [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
 ## Meetings
-* Working Group Meeting: TBD
-  * [Meeting notes and Agenda] TBD.
+
+* Working Group Meeting: monthly on the third Wednesday @ 11am Eastern Time ([Meeting Link](https://meet.google.com/pwb-trdw-tsq))
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1iY6pz8EtLhxabkwPmqsVQ1MOC7mZJSMuEjrbK7F0q6U/edit?usp=sharing).
 
 ## Organizers
 
 * Rebecca Whitworth
```
Comment (anishasthana) [inline wg-xai/README.md:24]: I can't say ive seen Rebecca around... is this list up to date?

File: wg-xai/README.md (hunk 1 above)
Comment (ruivieira) [reply wg-xai/README.md:26]: 👍 done!

**Review by LaVLaS — APPROVED** [review]

**Pull request merged**